var/spack/repos/builtin/packages/py-zfit/package.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyZfit(PythonPackage):
+    """
+    scalable pythonic model fitting for high energy physics
+    """
+
+    homepage = "https://github.com/zfit/zfit"
+    pypi = "zfit/zfit-0.18.0.tar.gz"
+
+    maintainers("jonas-eschle")
+    license("BSD-3-Clause", checked_by="jonas-eschle")
+
+    # TODO: reactivate once TF 2.15 is ready https://github.com/spack/spack/pull/41069
+    # version("0.18.1", sha256="fbc6b3a636d8dc74fb2e69dfec5855f534c4583ec18efac9e9107ad45b18eb43")
+    # version("0.18.0", sha256="21d9479480f74945c67707b715780693bd4e94062c551bf41fe04a2eddb47fab")
+    version("0.17.0", sha256="cd60dfc360c82666af4e8dddd78edb0ab95a095b9dd0868457f0981dc03afa5a")
+    version("0.16.0", sha256="b3b170af23b61d7e265d6fb1bab1d052003f3fb41b3c537527cc1e5a1066dc10")
+    version("0.15.5", sha256="00a1138429e8a7f830c9e229b9c0bcd6071b95dadd8c87eb81191079fb679225")
+    version("0.14.1", sha256="66d1e349403f1d6c6350138d0f2b422046bcbdfb34fd95453dadae29a8b0c98a")
+
+    depends_on("python@3.9:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+
+    variant("nlopt", default=False, description="Enable nlopt support")
+    variant("hs3", default=True, description="Enable serialization support")
+
+    with default_args(
+        type=("build", "run")
+    ):  # TODO: remove "build" once fixed in spack that tests need "run", not "build"
```

Comment:
If you put this comment on a different line you won't have to wrap the code on 3 lines